app/views/reports/regions/show.html.erb

```diff
@@ -74,12 +82,12 @@
                 <%= percentage_or_na(@data.last_value(:visited_without_bp_taken_rate), precision: 0) %>
               </p>
               <p class="flex-1 m-0px fs-14px">
-                Visited in the last 3 months but no BP measure
+                Visited in the last 3 months
                 <span
                   data-toggle="tooltip"
                   data-placement="top"
                   data-trigger="hover"
-                  title="Patients with an appointment scheduled, updated medicines, or blood sugar taken"
+                  title="Patients registered until the end of <%= @data[:adjusted_registrations].keys[-4] %> with an appointment scheduled, updated medicines, or blood sugar taken"
```

Comment:
Now that I'm reading this, we maybe shouldn't put the month here. It implies we are going back 3 three months from today for the _entire_ graph, but its going back 3 months for each month...so its a relative measure.  Maybe just clarify that in the titles?

Reply:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/69/90277770-c9462980-de2b-11ea-9983-052ea625205b.png">

The position of the tooltip implies it applies to the whole graph here...

Reply:
Great point @rsanheim! Updated it to read "Patients registered 3 months prior with an appointment scheduled..."